src/session/sidebar/chat_row.rs

```diff
@@ -236,6 +236,61 @@ fn stringify_message(message: Message) -> String {
         MessageContent::MessageSticker(data) => {
             format!("{} {}", data.sticker.emoji, gettext("Sticker"))
         }
+        MessageContent::MessagePhoto(data) => format!(
+            "{}{}",
+            gettext("Photo"),
+            if data.caption.text.is_empty() {
+                String::new()
+            } else {
+                format!(", {}", data.caption.text)
```

Comment:
The caption should be dimmed, just like the regular MessageText. Maybe adding a custom function to dim the label would be useful.

Reply:
Done. I put some functions in crate::utils that can be reused.
Additionally, I escaped the text, else it would not show up when it contains a special character.